specification/types.md

```diff
@@ -20,6 +20,10 @@ A numeric value of unspecified type or size. Implementation languages may furthe
 
 Structured data, presented however is idiomatic in the implementation language, such as JSON or YAML.
 
+### Date
+
+A language primitive for representing a date, possibly including timezone information.
```

Comment:
I would say it `MUST` include timezone information, otherwise it's just ambiguous. ISO8601 should be used.